File: exercicios/JavaScript/1133/1133.js
var input = require('fs').readFileSync('stdin','utf8');
var lines = input.split('\n');

var x = parseInt(lines.shift());
var y = parseInt(lines.shift());

var menor = x<y ? x : y;
var maior = x>y ? x : y;

for(let i=menor+1;i<maior; i++){
    if(i%5==2 || i%5==3){
        console.log(i);
    }
}

File: exercicios/R/1133/1133.r
input = file('stdin','r')

maior = as.integer(0)
menor = as.integer(0)

x = as.integer(readLines(input,n=1))
y = as.integer(readLines(input,n=1))
i = as.integer(0)
if(x > y){
    maior = x
    menor = y
}else{
    maior = y
    menor = x
}
i = menor + 1

while(i<maior){
    if(i%%5 == 2 || i%%5==3){
        write(i,'')
    }
    i= i + 1
}